process/working-group-lifecycle.md

```diff
@@ -1,19 +1,21 @@
-# Working Group Life Cycle
+# Working Group (WG) Life Cycle
 
-Contributors interested in starting a new working group are encouraged to first review the existing
-working groups and projects. It is very likely that an existing working group could be extended to
+Working Groups are under the governance of the Technical Advisory Committee (TAC).
+
+Contributors interested in starting a new WG are encouraged to first review the existing
+WGs and projects. It is very likely that an existing WG could be extended to
 cover the new topic. If this is the case, please bring your time and energy into that existing working
 group. Your contributions are very welcome!
```

Comment:
Nit: The paras in this document are wrapped instead of being single lines.

Reply:
Addressed in PR#117

Reply:
Since PR#117 is landed, should we update and resolve this?